src/trans/internal/sumplatb_mod.F90

```diff
@@ -94,7 +94,8 @@ SUBROUTINE SUMPLATB(KDGSA,KDGL,KPROCA,KLOENG,LDSPLIT,LDFOURIER,&
 INTEGER(KIND=JPIM) :: ICOMP, IGL, JA, JGL, ILAST, IREST, IA
 INTEGER(KIND=JPIM) :: ITOT_TOP, ITOT_BOT, IGL_TOP, IGL_BOT
 INTEGER(KIND=JPIB) :: IMEDIA,ITOT
-REAL(KIND=JPRB) :: ZLG
+
+!REAL(KIND=JPRB) :: ZLG
```

Comment:
Kept because referenced in below commented-out code?

Reply:
Correct